scripts/romanisim-make-stack

```diff
@@ -0,0 +1,267 @@
+                print(f'entry.keys = {entry.keys()}')
+
+            # Reset time offset
+            time_offset = 0 * u.s
+
+            # RA & Dec
+            coord = SkyCoord(ra=float(entry['ra']) * u.deg, dec=float(entry['dec']) * u.deg,
+                             frame='icrs')
+            wcs.fill_in_parameters(metadata, coord, boresight=args.boresight, roll_ref=float(entry['roll_angle']))
+
+            # Observational metadata
+            metadata['instrument']['optical_element'] = entry['optical_element']
+            metadata['exposure']['ma_table_number'] = int(entry['ma_table_number'])
+            ma_table = parameters.ma_table[ metadata['exposure']['ma_table_number']]
+
+
+
+
+            # Loop over number of exposures
+            for entry_idx in range(int(entry['exposures'])):
+                # Set initial (possibly only) SCA
+                sca = args.sca if (args.sca > 0) else 1
+
+                # Date
+                date = Time(entry['date'], format='isot') + time_offset
+                metadata['exposure']['start_time'] = date
+
+                # If SCA value is within 1-18, only run once
+                # otherwise loop over all detectors
+                while (sca <= NUMBER_OF_DETECTORS):
+                    # Detector
+                    metadata['instrument']['detector'] = f'WFI{sca:02d}'
+
+                    # Create output file name
+                    output_file_name = f"roman_simulated_{entry['ra']}_{entry['dec']}_{date}_{entry['optical_element']}_{entry_idx + 1}_{sca}.asdf"
+
+                    if args.debug:
+                        print(f"output_file_name = {output_file_name}")
+
+                    # If making a script, write the line and proceed
+                    # otherwise create simulation file
+                    if args.make_script:
+                        line = f"romanisim-make-image {output_file_name}"
+
+                        for option, value in options_dct.items():
+                            if (type(value) != bool) and (option != 'sca'):
+                                line += f" --{option} {value}"
+                            elif (option == 'sca'):
+                                line += f" --{option} {sca}"
+                            elif (value == True):
+                                line += f" --{option}"
+
+                        line += f" --bandpass {entry['optical_element']}"
+                        line += f" --radec {entry['ra']} {entry['dec']}"
+                        line += f" --ma_table_number {entry['ma_table_number']}"
+                        line += f" --date "
+                        for s in date.strftime('%Y %m %d %H %M %S %f').split(' '):
+                            line += f"{int(s)} "
+
+                        # print line
+                        if args.debug:
+                            print(f'line = {line}')
+
+                        # add line to script
+                        spcript_file.write(line+'\n')
```

Comment:
Two cents: I'd break this section out to a separate function outside the for entry in reader level.  This doesn't use anything else we've really computed above?